### DISCUSSION TITLE
Build Error

### DISCUSSION THREAD

**Discussion by BenHall-1** [webpack/webpack#14120] [Q&A]
# Bug report

**What is the current behavior?**
When we run a webpack build, it errors out on one specific file, however this has never errored before. 

**If the current behavior is a bug, please provide the steps to reproduce.**

Webpack Config: https://gist.github.com/BenHall-1/f055798f25db6e3daf4ff49bba58bc42

Error:
```log
ERROR in ./node_modules/cesium/Source/ThirdParty/zip.js 1565:71
    Module parse failed: Unexpected token (1565:71)
    You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
    | 	if (!workerData.interface) {
    | 		try {
    > 			workerData.worker = new Worker(new URL(workerData.scripts[0], import.meta.url));
    | 		} catch (error) {
    | 			workerData.worker = new Worker(new URL(workerData.scripts[0], import.meta.url), { type: "module" });
     @ ./node_modules/cesium/Source/DataSources/KmlDataSource.js 44:0-39 394:32-35 406:32-35 3168:19-22 3168:37-40
     @ ./node_modules/cesium/Source/Widgets/Viewer/viewerDragDropMixin.js
     @ ./node_modules/cesium/Source/Cesium.js
     @ ./src/index.js

```

Please note, it is not a bug with Cesium as we have not changed this in months, same with the webpack version, however it has just randomly\* stopped working (\*use of the word random is because there was no changes to the JS code prior to the builds starting to fail)

**What is the expected behavior?**
The build should complete without any errors

**Other relevant information:**
webpack version:  Tried on 4.44.2 and 5.51.2
Node.js version: 14.17.3
Operating System: Azure Pipeline (Ubuntu 20.04)
Additional tools:

**Comment by alexander-akait** [comment]
You still use `webpack` v4, webpack v4 doesn't support `import.meta.url`

**Reply by alexander-akait** [reply]
> https://gist.github.com/BenHall-1/f055798f25db6e3daf4ff49bba58bc42

Can you provide full steps to reproduce?
